pkg/trace/api/api.go

```diff
@@ -42,6 +43,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/trace/watchdog"
 )
 
+const outOfCPULogThreshold uint32 = 10
```

Comment:
Just curious: how / why did you pick 10? Also, you may want to document this.

Reply:
It's purely experimental / a tradeoff - see the discussion here https://github.com/DataDog/datadog-agent/pull/13268#discussion_r957476611
I'll add a comment in the code.

Reply:
Thank you for the context. That makes complete sense. Resolving.